### ISSUE TITLE
error when running vespa docker container on windows

### ISSUE THREAD

**Issue by MaazDev** [vespa-engine/vespa#25990]
**Describe the bug**
when I run the vespa docker container by running 
docker run `vespaengine/vespa` , I get the error `/usr/local/bin/start-container.sh: line 67:    10 Illegal instruction     /opt/vespa/bin/vespa-start-configserver`
**To Reproduce**
I followed the quickstart guide [here](https://docs.vespa.ai/en/vespa-quick-start.html)
1. made sure I had at least 4 gb memory 
2. `vespa config set target local`
3. 
```
docker run --detach --name vespa --hostname vespa-container \
  --publish 8080:8080 --publish 19071:19071 \
  vespaengine/vespa
```
4. watch the logs ur going to see `2023-02-10 22:36:30 /usr/local/bin/start-container.sh: line 67:    10 Illegal instruction     /opt/vespa/bin/vespa-start-configserver` with the message `EXITED (132)`

**Expected behavior**
the container should be running

**Screenshots**

**Environment (please complete the following information):**
 - OS: [e.g. iOS]: windows 10
 - self-hosted
 - Versions [e.g. 22]

**Vespa version**
Include Vespa version(s) used.
vespa:latest

**Additional context**

**Comment by baldersheim** [comment]
This sounds like a duplicate of #25892 
Will be fixed in first release upcoming week

**Comment by aressem** [comment]
Fix released in `8.128.22`. Latest is now `8.131.17`.
